### DISCUSSION TITLE
Not working anymore!

### DISCUSSION THREAD

**Discussion by XxHansYoloxX** [araghon007/X1nput#19] [General]
This worked fine about a year a ago, then a controller update came and now the impulse triggers do not work anymore.
Any fix incoming?

**Comment by araghon007** [comment]
Well, hopefully it'll work this time

**Comment by theAlekhnovich** [comment]
For now it is up and running on Windows 11. Checked with Enslave: Odessey to the West. Found this project on PC Gaming Wiki.

**Comment by Madrok9** [comment]
I've tried everything and I can't get X1nput to get impulse triggers to work in game! I've manually injected the file in games and still nothing. I'm using Steam on Windows 11.
